webapp/src/services/api/studies/links/index.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2024, RTE (https://www.rte-france.com)
+ *
+ * See AUTHORS.txt
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This file is part of the Antares project.
+ */
+
+import { StudyMetadata } from "@/common/types";
+import client from "../../client";
+import type { CreateLinkParams, DeleteLinkParams, LinkDTO } from "./types";
+
+export async function createLink(params: CreateLinkParams) {
+  const { studyId, ...body } = params;
+  const { data } = await client.post<LinkDTO>(
+    `/v1/studies/${params.studyId}/links`,
+    body,
+  );
+  return data;
+}
+
+export async function getLinks(params: { studyId: StudyMetadata["id"] }) {
+  const { data } = await client.get<LinkDTO[]>(
+    `/v1/studies/${params.studyId}/links`,
+  );
+  return data;
+}
+
+/**
+ * Deletes the link between the two specified areas.
+ *
+ * @param params - The parameters.
+ * @param params.studyId - The study ID.
+ * @param params.areaFrom - The from area name.
+ * @param params.areaTo - The to area name.
+ * @returns The deleted link id (format: `${areaFromId}%${areaToId}`)
+ */
+export async function deleteLink(params: DeleteLinkParams) {
+  const { studyId, areaFrom, areaTo } = params;
+  const { data } = await client.delete<string>(
+    `/v1/studies/${studyId}/links/${areaFrom}/${areaTo}`,
+  );
+  return data;
+}
```

Comment:
you can group all API calls into a single `linksApi` object, and destructure parameters directly in function arguments
```typescript
export const linksApi = {
  createLink: async ({ studyId, ...body }: CreateLinkParams) => {
    const { data } = await client.post<LinkDTO>(
      `/v1/studies/${studyId}/links`,
      body
    );
    return data;
  },

  getLinks: async ({ studyId }: { studyId: StudyMetadata["id"] }) => {
    const { data } = await client.get<LinkDTO[]>(
      `/v1/studies/${studyId}/links`
    );
    return data;
  },

  deleteLink: async ({ studyId, areaFrom, areaTo }: DeleteLinkParams) => {
    const { data } = await client.delete<string>(
      `/v1/studies/${studyId}/links/${areaFrom}/${areaTo}`
    );
    return data;
  }
};
```
Usage:
```typescript
import { linksApi } from './api/links/linksApi';

await linksApi.createLink({ studyId, areaFrom, areaTo });
await linksApi.getLinks({ studyId });
await linksApi.deleteLink({ studyId, areaFrom, areaTo });
```

Reply:
I don't like to force like this, with the standard export you can as you want:

`import { createLink, getLinks } from "..."`
`import * as linksApi from "..."`

And with an object it's difficult to combine function.